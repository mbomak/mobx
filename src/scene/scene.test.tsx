import React from 'react';
import {shallow} from 'enzyme';
import Home from './home';
import Registration from './registration';
import User from './user';

describe('Scene tests', () => {
    it('Should render Home without errors', () => {
        const wrapper = shallow(<Home />);
        const type: any = wrapper.type();
        expect(wrapper).toMatchSnapshot();
    });

    it('Should render Registration without errors', () => {
        const wrapper = shallow(<Registration />);
        const type: any = wrapper.type();
        expect(wrapper).toMatchSnapshot();
    });

    it('Should render User without errors', () => {
        const wrapper = shallow(<User />);
        expect(wrapper).toMatchSnapshot();
    });
});
