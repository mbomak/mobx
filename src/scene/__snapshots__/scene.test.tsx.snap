// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Scene tests Should render Home without errors 1`] = `ShallowWrapper {}`;

exports[`Scene tests Should render Registration without errors 1`] = `ShallowWrapper {}`;

exports[`Scene tests Should render User without errors 1`] = `ShallowWrapper {}`;
